qlib/data/storage/storage.py

```diff
@@ -0,0 +1,234 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from typing import Iterable, overload, Tuple, List, Text, Iterator, Union, Dict
+
+import pandas as pd
+
+# calendar value type
+CalVT = str
+
+# instrument value
+InstVT = List[Tuple[CalVT, CalVT]]
+# instrument key
+InstKT = Text
+
+
+class CalendarStorage:
+    def __init__(self, freq: str, future: bool, uri: str):
```

Comment:
Add documents to explain that all behaviors follow the definition of List by default.